docs/conceptual/interfaces-[fsharp].md

```diff
@@ -47,6 +47,12 @@ member-list
 ## Remarks
 Interface declarations resemble class declarations except that no members are implemented. Instead, all the members are abstract, as indicated by the keyword **abstract**. You do not provide a method body for abstract methods. However, you can provide a default implementation by also including a separate definition of the member as a method together with the **default** keyword. Doing so is equivalent to creating a virtual method in a base class in other .NET languages. Such a virtual method can be overridden in classes that implement the interface.
 
+You can optionally give each method parameter a name using normal F# syntax:
+
+[!code-fsharp[Main](snippets/fslangref1/snippet24032.fs)]
+
+In the above **ISprintable** example, the **Print** method has a single parameter of the type **string** with the name *format*.
```

Comment:
Can you please wrap the code names (e.g. `ISprintable`) in single quotes instead of bold?